contrib/systemd/origin-node.sysconfig

```diff
@@ -13,10 +13,3 @@ CONFIG_FILE=/etc/origin/node/node-config.yaml
 # TODO: More elegant solution like this
 # https://github.com/coreos/flannel/blob/master/dist/mk-docker-opts.sh
 # DOCKER_NETWORK_OPTIONS='-b=lbr0 --mtu=1450'
-
-# Proxy configuration
```

Comment:
Do these affect any of the calls the node makes using the docker client or network calls from inside containers?